dags/ods/post-event-report/scripts/trigger-report-generator.sh

```diff
@@ -0,0 +1,25 @@
+dir_2020_postevent_data="${HOME}/pycontw-report-generator-data"
+
+csv_talks="${dir_2020_postevent_data}/misc/talks-list.csv"
```

Comment:
Should we remove the 2020 sub string to make it more general?

Reply:
And I'm curious about how these data/file should be placed or loaded.

Reply:
+1 to removal the 2020 prefix

data/file loaded implementation: this is also called "phase 2" of implementation, and it will be very like loaded via Bigquery, and the solution is still an open topic. Inputs are very welcome.